### PR TITLE
fix: activities pagination

### PR DESCRIPTION
**Pull request by vatanasov**
refs: https://github.com/aeternity/ae_mdw/issues/1883

### CODE REVIEW

File: lib/ae_mdw/activities.ex [hunk 1]
```diff
@@ -527,11 +522,11 @@ defmodule AeMdw.Activities do
           nil ->
             nil
 
-          {txi_cursor, local_idx_cursor} when direction == :forward ->
-            {account_pk, pos, txi_cursor, local_idx_cursor}
+          {txi_cursor, _local_idx_cursor} when direction == :forward ->
+            {account_pk, pos, txi_cursor, @min_int}
```
Comment (sborrazas) [inline lib/ae_mdw/activities.ex:526]: Why is this local cursor ignored? What happens when there's more than 10 int calls in the same txi? Wouldn't the pagination loop infinitely here?

Comment (vatanasov) [reply lib/ae_mdw/activities.ex:526]: It shouldn't. This `local_idx` works when there are only int calls with the same `txi`, because it corresponds to the supporting idx assigned at https://github.com/aeternity/ae_mdw/blob/master/lib/ae_mdw/activities.ex#L684. But when there are other types of activities with the same txi, they are now basically different variables. For example, if we have a `NameClaimEvent` and two internal calls (with their internal `local_idx` being 0 and 1), at https://github.com/aeternity/ae_mdw/blob/master/lib/ae_mdw/activities.ex#L684 they will get assigned 0 for the name claim, 1 for the first internal call (even though his local idx is 0) and 2 for the second internal call (even though his local idx is 1). That means, that if we paginate by 1, the first page will be fine, but on the second page (where the cursor is at `local_idx` 1), it will skip over the name claim (at https://github.com/aeternity/ae_mdw/blob/master/lib/ae_mdw/activities.ex#L212) and on the int call stream it will skip over the first int call.

Basically we have a global `local_txi`, that is assigned to all the type of activities, that is being compared to the local `local_txi` for the int calls, and in case there is more than one type of activity for a given txi, this leads to skipping on pagination, because they don't match.

I am not sure if the provided explanation is clear enough, feel free to say so and I will try to provide a more descriptive one.

Comment (sborrazas) [reply lib/ae_mdw/activities.ex:526]: I understand how activities work, I've implemented it and the concept of using it to list activities of the same transaction. My question was the following: if we are using local_idx in the cursor to skip the first N results of things we've already seen, why are we ignoring it?

Comment (sborrazas) [reply lib/ae_mdw/activities.ex:526]: Nvm, I was wrong, we're using another index for txi activities. Thank you for explaining

**Review by sborrazas — APPROVED** [review]
